google/cloud/pubsub/subscriber.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/pubsub/subscriber.h"
+
+namespace google {
+namespace cloud {
+namespace pubsub {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+
+Subscriber::Subscriber(std::shared_ptr<SubscriberConnection> connection,
+                       Options opts)
+    : connection_(std::move(connection)),
+      options_(google::cloud::internal::MergeOptions(std::move(opts),
```

Comment:
Q.  Do we like `google::cloud::` qualifications when we're already in `google::cloud`?

Reply:
Fixed.